### ISSUE TITLE
Redundant bottom space 

### ISSUE THREAD

**Issue by closetoyou293** [matteobortolazzo/HtmlLabelPlugin#10]
![2018-01-19_11-12-04-am](https://user-images.githubusercontent.com/19910414/35134580-bb349ada-fd09-11e7-8b3a-abaabd412cd5.jpg)

There is redundant space at bottom.
It didn't happen in previous build.
It seems lib adds more extra row.

**Comment by matteobortolazzo** [comment]
Can you provide me an example, please?

**Comment by matteobortolazzo** [comment]
Ok It's an Android issue and there was also in the previous build because it's the Html.FromHtml method... It adds a "\n" every time. BTW I solved it

**Comment by matteobortolazzo** [comment]
Fixed in [2.1.1](https://www.nuget.org/packages/Xam.Plugin.HtmlLabel/2.1.1)

**Comment by primehalo** [comment]
I know you say this was fixed in 2.1.1, but did it broken again in 3.0.1.3-beta because I'm seeing this behavior in iOS. In Android there is no blank space at the bottom.

**Comment by matteobortolazzo** [comment]
Sorry, I solved it only on Android, I didn't notice it was also on iOS. Unfortunately, I don't have an iOS device now... Is there a way you can find the issue and do a pull request? Or tells me where the problem is at least?

**Comment by primehalo** [comment]
I had found a workaround which was to force 1 line, `htmlLabel:HtmlLabel.MaxLines="1"`, even though the strings were only a single anyway. After I finish my project I'll try to come back and take a look at this.

**Comment by matteobortolazzo** [comment]
Thanks @primehalo :) I hope to find an iPhone to debug it myself

**Comment by Axort** [comment]
I was experiencing this with iOS and I already found what is wrong (sort of). In CreateAttributedString method, when the NSMutableAttributedString is created, it creates it with a \n in the end of the string (I don't know why). What I did was, after creating the NSMutableAttributedString...:

```
var mutable = new NSMutableAttributedString(new NSAttributedString(myHtmlData, attr, ref nsError));
if(mutable.MutableString.HasSuffix(new NSString("\n"))) {
    mutable.DeleteRange(new NSRange(mutable.MutableString.Length - 1, 1));
}
```

This solves the issue but I don't know if it could affect another scenario. I hope it helps.
